release-pipeline.yml

```diff
@@ -1710,7 +1711,7 @@ stages:
                     $(Build.ArtifactStagingDirectory)/azCopy-mac-signed/*
                     $(Build.ArtifactStagingDirectory)/azCopy-windows-signed/*
                   changeLogCompareToRelease: 'lastFullRelease'
-                  changeLogType: 'commitBased'
+                  changeLogFile: '$(Build.SourcesDirectory)/ChangeLog.md'
```

Comment:
This will upload the changelog.md to your github release which is not correct. Ideally it shall only get you the commit history or only the part from Changelog.md for this release and not everthing.

Reply:
changing back to commitBased